wp-content/themes/borderzine/less/inc/header.less

```diff
@@ -17,4 +17,41 @@
 #main-nav.navbar {
 	border-top: none;
 	border-bottom: 2px solid @orange;
-}
\ No newline at end of file
+}
+.subscribe-btn {
+    float: left;
+    margin: 5px 0 0 10px;
+    background: @white;
+    font-size: 14px;
+    line-height: 2;
+    a {
+        padding: 24px 7px;
+        color: @orange;
+        &:hover {
+            text-decoration: none;
+            color: @orange;
+        }
+    }
+    &:hover {
+        background: darken(@white, 5%);
```

Comment:
I'm worried that this background-color darkening is not enough to provide a legible darkening. Can it go darker?

Reply:
Sure! Let's try 10%

Reply:
Updated to 10% in https://github.com/INN/umbrella-borderzine/pull/84/commits/9835e2a610507b1e052cdb24c4cd4bbaf12ceb89